src/services/study/dynamic-security-analysis.ts

```diff
@@ -15,27 +15,25 @@ import {
 } from './dynamic-security-analysis.type';
 import { fetchContingencyAndFiltersLists } from '../directory';
 
-const PREFIX_DYNAMIC_SECURITY_ANALYSIS_SERVER_QUERIES = import.meta.env.VITE_API_GATEWAY + '/dynamic-security-analysis';
-
-function getDynamicSecurityAnalysisUrl() {
-    return `${PREFIX_DYNAMIC_SECURITY_ANALYSIS_SERVER_QUERIES}/v1/`;
-}
+export function startDynamicSecurityAnalysis(
+    studyUuid: UUID,
+    currentNodeUuid: UUID,
+    currentRootNetworkUuid: UUID,
+    debug: boolean
+): Promise<void> {
+    console.info(`Running dynamic security analysis on '${studyUuid}' and node '${currentNodeUuid}' ...`);
```

Comment:
It's just a detail but it looks like now currentRootNetworkUuid is logged as well : 

```suggestion
    console.info(`Running dynamic security analysis on '${studyUuid}', on root network '${currentRootNetworkUuid}' and node '${currentNodeUuid}' ...`);
```